Fix "Classes/ToolBox.cpp:29:5: error: use of undeclared identifier 'usleep'"
--- Classes/ToolBox.h.orig	2009-03-22 12:29:23.000000000 -0500
+++ Classes/ToolBox.h	2018-11-07 12:11:09.000000000 -0600
@@ -19,6 +19,7 @@
 #ifndef TOOLBOX_H
 #define TOOLBOX_H
 
+#include <unistd.h>
 #include <QString>
 #include <QIcon>
 #include "../global.h"
